manageiq-operator/pkg/apis/manageiq/v1alpha1/manageiq_types.go

```diff
@@ -0,0 +1,116 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// ManageiqSpec defines the desired state of Manageiq
+// +k8s:openapi-gen=true
+type ManageiqSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
+	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
+	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+	// Application name used for deployed objects
+	AppName string `json:"appName"` 
+
+	// admin user initial password
+	ApplicationAdminPassword string `json:"applicationAdminPassword"` 
+
+	// Domain name for the external route
+	// Used for external authentication configuration
+	ApplicationDomain string `json:"applicationDomain"`
+
+	DatabaseName string `json:"databaseName"`
+	DatabasePort string `json:"databasePort"`
+	DatabaseUser string `json:"databaseUser"`
+	DatabasePassword string `json:"databasePassword"`
+	// Application region number
+	DatabaseRegion string `json:"databaseRegion"`
+	// Containerized database volume size
+	DatabaseVolumeCapacity string `json:"databaseVolumeCapacity"`
+
+	HttpdCPUReq string `json:"httpdCPUReq"`
+	HttpdImageName string `json:"httpdImageName"`
+	HttpdImageTag string `json:"httpdImageTag"`
+	HttpdMemLimit string `json:"httpdMemLimit"`
+	HttpdMemReq string `json:"httpdMemReq"`
```

Comment:
@Fryguy do we want to take this opportunity to not abbreviate unnecessarily?

I'm thinking `...MemoryRequest` is not too much, right?

Reply:
👍 for not abbreviating...though we can probably do that in a followup